src/styles/sidebar/components/group-list.scss

```diff
@@ -21,6 +21,27 @@ $group-list-spacing-below: 50px;
     }
   }
 
+  .dropdown-menu__section--no-header {
+    border-top: solid 1px rgba(0, 0, 0, 0.15);
+
+    .group-details {
+      font-weight: 400;
+    }
+  }
```

Comment:
I see this CSS is attempting to fit into the existing structure. However, the existing structure doesn't follow our conventions well as it was written some time ago.

The preferred pattern is that each `styles/components/{component-name}.scss` file defines one "top level" class and its subsections and variants (each CSS rule is of the form `.{component-name}__{subcomponent}--{variant}` where the latter two parts are optional. This makes it easy to know what file a particular component is found in.

Each UI component (whether written in Angular or React) will then have one associated style component of the same name. There will also be style components that don't have a corresponding Angular component (because they are just components that just need styling and no complex markup or controller logic).

As long as all the new styles defined here are only used in the `<group-list>` component then I think we can leave this as-is and improve the situation when we come to migrate this to React.